adoc/chapters/opencl_backend.adoc

```diff
@@ -443,13 +443,6 @@ whether the object has actively been used on these devices yet or not.
 
 // From 3.10 Language restrictions in kernels
 
-Some types in SYCL vary according to pointer size or vary on the host
-according to the host ABI, such as [code]#size_t# or [code]#long#. In order
-for the SYCL device compiler to ensure that the sizes of
-these types match the sizes on the host and to enable data of these types
-to be shared between host and device, the OpenCL interoperability types
-are defined, [code]#sycl::cl_int# and [code]#sycl::cl_size_t#.
```

Comment:
This section does not seems wrong as an introductory matter, besides the `size_t` and the name space.
So why not just replacing `size_t` by `short` for example and replace the `sycl::cl_` by `sycl::opencl::cl_`?

Reply:
To me, it seems like this whole paragraph is talking about cases where a type might have a different size in host vs. device.  I think the first sentence uses `long` as an example because that could be 32-bits on some systems and 64-bits on others.  The implication is that the host and device might have different interpretations about the size of `long`.  However, this is not possible in SYCL because we mandate that all fundamental types have the same size on host and device.